src/averageAges.js

```diff
@@ -36,8 +41,23 @@ function calculateMenAverageAge(people, century) {
  *
  * @return {number}
  */
+
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  let female = withChildren
+    ? people
+      .filter(person => person.mother !== null)
+      .map(person => person.mother)
+    : people.filter(person => person.sex === 'f');
+
+  female = withChildren
+    ? people.filter(item => female.includes(item.name))
+    : female;
+
```

Comment:
You might called it woman or female, its not item, lol
```suggestion

  female = withChildren
    ? people.filter(woman => female.includes(woman.name))
    : female;

```